src/main/java/no/training/project/resource/version/Version.java

```diff
@@ -0,0 +1,54 @@
+package no.training.project.resource.version;
+
+import jakarta.ws.rs.*;
+import jakarta.ws.rs.core.MediaType;
+import no.training.project.Mapper.ReleaseMapper;
+import no.training.project.exception.ServiceException;
+import no.training.project.model.Release;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+@Path("/release-name")
+public class Version {
```

Comment:
Since this is not a model class, but a class serving endpoints, it would be better to call it `VersionResource`.

Reply:
Fixed